cmd/fleetctl/apple_mdm.go

```diff
@@ -178,126 +181,70 @@ Next, use the generated certificates to deploy Fleet with `+"`mdm`"+` configurat
 	}
 }
 
-func appleMDMSetupCommand() *cli.Command {
-	return &cli.Command{
-		Name:  "setup",
-		Usage: "Setup commands for Apple MDM",
-		Subcommands: []*cli.Command{
-			appleMDMSetupDEPCommand(),
-		},
-	}
-}
-
-func appleMDMSetupDEPCommand() *cli.Command {
-	return &cli.Command{
-		Name:  "dep",
-		Usage: "Configure DEP token",
-		Subcommands: []*cli.Command{
-			appleMDMSetDEPTokenInitCommand(),
-			appleMDMSetDEPTokenFinalizeCommand(),
-		},
-	}
-}
-
-func appleMDMSetDEPTokenInitCommand() *cli.Command {
+func generateMDMAppleBMCommand() *cli.Command {
 	return &cli.Command{
-		Name:  "init",
-		Usage: "Start DEP token configuration",
+		Name:    "mdm-apple-bm",
+		Aliases: []string{"mdm_apple_bm"},
+		Usage:   "Generate Apple Business Manager public and private keys to enable automatic enrollment for macOS hosts.",
 		Flags: []cli.Flag{
-			configFlag(),
-			contextFlag(),
+			&cli.StringFlag{
+				Name:  "public-key",
+				Usage: "The output path for the Apple Business Manager public key certificate.",
+				Value: bmPublicKeyCertPath,
+			},
+			&cli.StringFlag{
+				Name:  "private-key",
+				Usage: "The output path for the Apple Business Manager private key.",
+				Value: bmPrivateKeyPath,
+			},
 		},
 		Action: func(c *cli.Context) error {
-			const (
-				cn = "fleet"
-				// Setting validityDays to 10 in case user doing the init command
-				// is different than user uploading to Apple.
-				// (Though we've heard from other users that Apple doesn't really check
-				// the expiration of this public key.)
-				validityDays = 10
-				pemCertPath  = "fleet-mdm-apple-dep.crt"
-				pemKeyPath   = "fleet-mdm-apple-dep.key"
+			publicKeyPath := c.String("public-key")
+			privateKeyPath := c.String("private-key")
+
+			cn := "FleetDM"
+
+			// Apple doesn't check the expiry
+			validityDays := 30
```

Comment:
Same here, `const`. It's not clear to me what the comment above is supposed to convey? It seems like it's missing a part that says why this matters (e.g. `Apple doesn't check the expiry, so we need to do x y and z`?).